src/NexusMods.App.UI/Helpers/StringFormatters.cs

```diff
@@ -0,0 +1,43 @@
+namespace NexusMods.App.UI.Helpers;
+
+/// <summary>
+/// Class that helps you format file sizes as strings.
+/// </summary>
+public static class StringFormatters
+{
+    /// <summary>
+    /// Formats the file size as a string.
+    /// </summary>
+    /// <param name="currentBytes">The currently downloaded number of bytes.</param>
+    /// <param name="totalBytes">The total number of downloaded bytes.</param>
+    /// <remarks>Complete number of downloaded bytes.</remarks>
+    public static string ToGB(long currentBytes, long totalBytes)
```

Comment:
We already use the `Humanizer` library for this

Reply:
Working as intended. 

`Humanizer` doesn't have an extension for formatting progress in format `current / total`. There is one for throughput, e.g. `1MB/s`, but not for progress.

The format shown in UI designs is `0.00 GB / 0.00 GB`. 
(Although it doesn't explicitly state whether it's `0.##` or `0.00`)

Humanizer formats as `0.##` (not customizable), so `500MB / 1000MB` would be formatted as `0.5 GB / 1 GB`.
(using `$"{currentBytes.Bytes().Humanize("GB")} GB / {totalBytes.Bytes().Humanize("GB")}"`)

This is undesirable, as the width of the download progress text would change as digits get added/removed in the rare random cases where progress represents a nice, round number for 1 second. After that 1 second, text would return to previous size, making the experience awkward.

If you instead use format `$"{currentBytes.Bytes().Humanize("0.00")} GB / {totalBytes.Bytes().Humanize("0.00")}"`, Humanizer will round to integers, in which case you will get inaccurate values (i.e. 0GB when 400+MB downloaded).

The current code ensures the download progress is formatted consistently.

Reply:
We have quite a bit of flexability on code like this, there's nothing that says we have to match the mockups 100%

Reply:
Yeah, though in this situation it would just be awkward if all the test is aligned 99% of the time and 1% of the time it becomes not aligned for 1 second. That's why I decided to go with this approach, it would be unsightly for the eyes.

This PR has actually already done this sort of thing with merging the pause/resume and progress bar column together, as we discussed and decided it would lead to better UX. (Happened when you were vacationing)

Reply:
Can I request that https://github.com/Nexus-Mods/NexusMods.App/pull/360/commits/d1222c0edb79e13a4c0e0b841687fb8b5691d737 gets feedback from an extra set of eyeballs?

The change in this commit will lead to strings such as `677.11 MB / 953.67 MB` to be printed.

The spacing in the DataGrid for InProgress downloads is already *very tight* as is. There is not a lot of horizontal space available, and as such, this change to formatting will lead to excessive text clipping; within that menu. That's another factor that has to be considered here.

We can save some space by trimming the `Version` column [down to around 80], as it's probably the thing that end users will care about the least; and anything beyond a standard semver length string is redundant.

Extra Note: No progress will display as `0 b` in Humanizer, which means you can get a string such as `0 b / 2.58 GB`. Might want to special case this one.